types/node/index.d.ts

```diff
@@ -26,12 +26,18 @@
 //                 Lishude <https://github.com/islishude>
 //                 Andrew Makarov <https://github.com/r3nya>
 //                 Zane Hannan AU <https://github.com/ZaneHannanAU>
+//                 Thomas den Hollander <https://github.com/ThomasdenH>
 //                 Eugene Y. Q. Shen <https://github.com/eyqs>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
 /** inspector module types */
 /// <reference path="./inspector.d.ts" />
 
+// Forward-declare SharedArrayBuffer (available starting es2017)
+interface SharedArrayBuffer {
```

Comment:
Please move this down to other forward declarations (`Iterable<T>`,...).

Reply:
Done.